.github/container/install-pax.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+## Parse command-line arguments
+
+usage() {
+    echo "Usage: $0 [OPTION]..."
+    echo "  -d, --dir=PATH         Path to store Pax source. Defaults to /opt"
+    echo "  -f, --from_paxml=URL   URL of the Paxml repo. Defaults to https://github.com/google-research/paxml.git"
+    echo "  -g, --from_praxis=URL  URL of the Praxis repo. Defaults to https://github.com/google-research/praxis.git"
+    echo "  -h, --help             Print usage."
+    echo "  -r, --ref_paxml=REF    Git commit hash or tag name that specifies the version of Paxml to install. Defaults to HEAD."
+    echo "  -q, --ref_praxis=REF   Git commit hash or tag name that specifies the version of Praxis to install. Defaults to HEAD."
+    exit $1
+}
+
+args=$(getopt -o d:f:hr: --long dir:,from:,help,ref: -- "$@")
```

Comment:
This needs to be updated in accordance with the usage info above. For example, for the `-d, --dir` option, we need to add `-o d:` and `--long dir:`. The trailing colon is only added if an option takes an argument.